spec/data-model.md

```diff
@@ -145,17 +145,22 @@ interface Option {
 }
 ```
 
-A `Reserved` represents an _expression_ with a _reserved_ _annotation_.
-The `sigil` corresponds to the starting sigil of the _reserved_.
+A `Reserved` represents an _expression_ with a _reserved_ or _private-use_ _annotation_.
+The `sigil` corresponds to the starting sigil of the _annotation_.
 The `source` is the "raw" value (i.e. escape sequences are not processed)
-and includes the starting `sigil`.
+and does not include the starting `sigil`.
 
 Implementations MUST NOT rely on the set of `sigil` values remaining constant,
 as future versions of this specification MAY assign other meanings to such sigils.
```

Comment:
I think this normative text is unenforceable. If I hardcoded the existing sigils for functions, pua, and reserved into my code, that's my business. I think any test we make of an MFv2 "1.0" implementation would absolutely depend on implementations treating the set of sigil values as constant (for that version).

I think what you mean is:

```suggestion
> **Note**
> Implementers should be aware that future versions of this specification
> might assign meaning to _reserved_ `sigil` values.
```

PS> I use a non-normative "should" because Notes are non-normative.

Reply:
Sure; the MUST NOT is indeed probably pretty impossible to break. I think the note might read better with this expansion?

```suggestion
> **Note**
> Implementers should be aware that future versions of this specification
> might assign meaning to _reserved_ `sigil` values
> and add new interfaces to this data model.
```

Reply:
Yes, that.